spec/lib/efile/md/md502_calculator_spec.rb

```diff
@@ -936,6 +938,7 @@
       create(
         :state_file_md_intake,
         filing_status: filing_status,
+        spouse_birth_date: MultiTenantService.statefile.end_of_current_tax_year - 40,
```

Comment:
not my favorite solution :/

Reply:
https://github.com/codeforamerica/vita-min/pull/5512/commits/f336cb7f8ffc9dd055b1f6bf2e6accab56af503a
opted to using `with_spouse` instead of hard-coding bday. The qss xml will still be used (overwrites factory xml)